c/ldv-races/race-3_2-container_of-global.i

```diff
@@ -294,14 +292,14 @@ typedef unsigned int pthread_key_t;
 typedef int pthread_once_t;
 union pthread_attr_t
 {
-  char __size[36];
+  char __size[56];
```

Comment:
Is this change related to machine model or other changes in C-Pthread header files?